infinistore/test_infinistore.py

```diff
@@ -42,7 +42,11 @@ def get_gpu_count():
 @pytest.mark.parametrize("local", [True, False])
 def test_basic_read_write_cache(server, dtype, new_connection, local):
     config = infinistore.ClientConfig(
-        host_addr="127.0.0.1", service_port=22345, dev_name="mlx5_0"
+        host_addr="127.0.0.1",
+        service_port=22345,
+        dev_name="mlx5_1",
```

Comment:
unit test is not testing on YOUR NODE. the parameter should be general and could work on most hardware-compatible node. 
Since infinistore coudl find ACTIVE device, unit test should automatically find avaliable mlx device

Reply:
ok, just updated, removed dev_name, link_type and ib_port